view/theme/frio/templates/field_checkbox.tpl

```diff
@@ -1,7 +1,7 @@
 
 	<div class="field checkbox" id="div_id_{{$field.0}}">
 			<input type="hidden" name="{{$field.0}}" value="0">
-			<input type="checkbox" name="{{$field.0}}" id="id_{{$field.0}}" value="1" {{if $field.2}}checked="checked"{{/if}} aria-checked="{{if $field.2}}true{{else}}false{{/if}}" aria-describedby="{{$field.0}}_tip" {{if $field.4}}{{$field.4}}{{/if}}>
+			<input type="checkbox" name="{{$field.0}}" id="id_{{$field.0}}" value="1" {{if $field.2}}checked="checked"{{/if}} aria-checked="{{if $field.2}}true{{else}}false{{/if}}" {{if $field.4}}aria-describedby="{{$field.0}}_tip"{{/if}} {{if $field.4}}{{$field.4}}{{/if}}>
```

Comment:
Shouldn't it be `{{if $field.3}}aria-describedby="{{$field.0}}_tip"{{/if}} `?

Reply:
Yeah, seems as if.